spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -27,6 +27,10 @@ private SparkSQLProperties() {}
   // Controls whether vectorized reads are enabled
   public static final String VECTORIZATION_ENABLED = "spark.sql.iceberg.vectorization.enabled";
 
+  // Controls which Parquet reader to use for vectorization
```

Comment:
Not necessarily vectorization, right? What about `Controls which Parquet reader implementation to use`?

Reply:
Fixed